src/status_im/contexts/chat/events.cljs

```diff
@@ -144,19 +144,21 @@
 
 (rf/defn close-chat
   {:events [:chat/close]}
-  [{:keys [db] :as cofx}]
-  (when-let [chat-id (:current-chat-id db)]
-    (chat.state/reset-visible-item)
-    (rf/merge cofx
-              {:db                        (-> db
-                                              (dissoc :current-chat-id)
-                                              (assoc-in [:chat/inputs chat-id :focused?] false))
-               :effects.async-storage/set {:chat-id nil
-                                           :key-uid nil}}
-              (link-preview/reset-all)
-              (delete-for-me/sync-all)
-              (delete-message/send-all)
-              (offload-messages chat-id))))
+  [{:keys [db] :as cofx} chat-id]
+  (let [current-chat-id (:current-chat-id db)
+        chat-id         (or chat-id current-chat-id)]
```

Comment:
I checked that `:chat/close` from `:pop-to-root` is calling first even if we are navigating to community chat from universal link. This is just a precautionary step to avoid silly race condition bugs.